templates/Default/engine/Default/planet_construction.php

```diff
@@ -1,4 +1,4 @@
-<p>You are currently building: <?php
+<p><span name="currently_building">You are currently building: <?php
```

Comment:
I expect the issue is that because a `<div>` is not a valid element to be contained within a `<p>` the html parser automatically ends the `<p>` when it sees the `<div>`.  It probably makes sense to drop the `<p>` altogether